### PR TITLE
[openuv] Fix openUV addon name and installation instructions

### PR DESCRIPTION
**Pull request by rudelm**
Hi,

I've tried yesterday to install the openUV addon. I could not find the addon from the list of add-ons and according to the [current documentation](https://www.openhab.org/addons/bindings/openuv/) it should be sufficient to just add the Bridge and the Items and the addon is configured.

The values were still empty and I found this discussion which pointed me into the right direction:
https://community.openhab.org/t/solved-location-of-openuv-binding/49200/3

The plugin can be found if you search for `opensprinkler`. Therefore it seems to be a copy&paste error when the `feature.xml` was touched. 

Here's a screenshot of how it currently looks in openHAB 2.4:

openuv-opensprinkler

I've updated now the `feature.xml` to show the correct name of the addon "openUV" and also updated the binding's `readme.md` to make the installation instruction clear.

I assume I don't need to sign off the changes, since it is a spelling correction as well as changes to the documentation, please correct me if I'm wrong.

### CODE REVIEW

**Comment by rudelm** [comment]
Seems that I'm unable to sign-off the first commit? The sign-off with amend showed both commit log lines so I assumed it would combine them... what should I do to fix that condition so the check will pass?

The Build failed, but seemingly for other issues?

**Comment by Hilbrand** [comment]
Or squash the commits or reword. You can use git rebase:
- `git rebase -i HEAD~2`
- change `pick` of the first commit message into `reword` or `pick` of the second commit in `squash`.
- `Ctrl-O` to write, and `Ctrl-X` to save. It should show an editor were you can update the commit message.
- Than push force: `git push --force-with-lease` to push the commit to the pull request.

**Comment by rudelm** [comment]
> Or squash the commits or reword. You can use git rebase:
> 
> * `git rebase -i HEAD~2`
> * change `pick` of the first commit message into `reword` or `pick` of the second commit in `squash`.
> * `Ctrl-O` to write, and `Ctrl-X` to save. It should show an editor were you can update the commit message.
> * Than push force: `git push --force-with-lease` to push the commit to the pull request.

Thanks @Hilbrand I think I've got the other commit message now also signed off :)

File: addons/binding/org.openhab.binding.openuv/README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
-# Open UV Binding
+# OpenUV Binding
 
 This binding uses the [OpenUV Index API service](https://www.openuv.io/) for providing UV Index information for any location worldwide.
 
 To use this binding, you first need to [register and get your API token](https://www.openuv.io/auth/google).
 
-## Supported Things
+## Binding Installation
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.openuv/README.md]: Please add an empty line after the headline and put each sentence I to a separate line.

Comment (rudelm) [reply addons/binding/org.openhab.binding.openuv/README.md]: ok done

**Comment by clinique** [comment]
Thanks a lot, I've been searching the source of this error in the OpenUV folder, without having found it. Good job !

**Comment by rudelm** [comment]
You're welcome :) Thanks for merging this so quick! I guess it will be then available in the next openHAB update 2.5?

**Pull request merged**